sandbox/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+project(Sandbox)
+
+find_package(
+  Qt5
+  COMPONENTS Core Quick QuickControls2
+  REQUIRED)
+
+add_executable(
+  ${PROJECT_NAME}
+  main.cpp
+  handler.cpp
+  sandboxapp.cpp
+  spellchecker.cpp
+  qml.qrc
+  ../src/statusq.qrc
+  ../src/assets.qrc)
+
+if(APPLE)
+  target_sources(${PROJECT_NAME} PRIVATE statuswindow_mac.mm)
+else()
+  target_sources(${PROJECT_NAME} PRIVATE statuswindow.cpp)
+endif()
+
+target_compile_definitions(${PROJECT_NAME}
+                           PRIVATE SRC_DIR="${CMAKE_CURRENT_LIST_DIR}")
+target_link_libraries(
+  ${PROJECT_NAME} PRIVATE Qt5::Core Qt5::Quick Qt5::QuickControls2
+                          SortFilterProxyModel)
```

Comment:
You should also preserve these from the qmake file:
```
QML_IMPORT_PATH= $$PWD/../src

DEFINES += QT_DEPRECATED_WARNINGS

QMAKE_LFLAGS += $$(LDFLAGS)
```

Reply:
For the cmake/QML_IMPORT_PATH trick, see: https://stackoverflow.com/a/38266179/2248784